[2022-07-14 14:07:41,862] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.process_user manual__2022-07-14T14:07:34.490179+00:00 [queued]>
[2022-07-14 14:07:41,867] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.process_user manual__2022-07-14T14:07:34.490179+00:00 [queued]>
[2022-07-14 14:07:41,869] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-14 14:07:41,870] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-14 14:07:41,871] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-14 14:07:41,878] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): process_user> on 2022-07-14 14:07:34.490179+00:00
[2022-07-14 14:07:41,884] {standard_task_runner.py:52} INFO - Started process 8185 to run task
[2022-07-14 14:07:41,887] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'process_user', 'manual__2022-07-14T14:07:34.490179+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpg1m7zmgg', '--error-file', '/tmp/tmp04zegejm']
[2022-07-14 14:07:41,889] {standard_task_runner.py:80} INFO - Job 28: Subtask process_user
[2022-07-14 14:07:41,932] {task_command.py:369} INFO - Running <TaskInstance: user_processing.process_user manual__2022-07-14T14:07:34.490179+00:00 [running]> on host 9d087153d2d5
[2022-07-14 14:07:41,983] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=process_user
AIRFLOW_CTX_EXECUTION_DATE=2022-07-14T14:07:34.490179+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-14T14:07:34.490179+00:00
[2022-07-14 14:07:42,004] {python.py:173} INFO - Done. Returned value was: None
[2022-07-14 14:07:42,014] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=process_user, execution_date=20220714T140734, start_date=20220714T140741, end_date=20220714T140742
[2022-07-14 14:07:42,064] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-14 14:07:42,094] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
